pythainlp/corpus/corpus_license.md

```diff
@@ -101,3 +101,34 @@ S. Thoongsup et al., ‘Thai WordNet construction’,
 in Proceedings of the 7th Workshop on Asian Language Resources,
 Suntec, Singapore, Aug. 2009, pp. 139–144.
 https://www.aclweb.org/anthology/W09-3420.pdf 
+
+## Wikipedia Titles
+Corpus of Wikipedia titles (wikipedia_titles.txt) was processed by konbraphat51 (https://github.com/konbraphat51/Thai_Dictionary_Cleaner/tree/main)
+
+The original data is thwiki-latest-all-titles.gz of https://dumps.wikimedia.org/thwiki/latest/ which Wikipedia.org has created.
```

Comment:
Do you mind to add date of the Wikipedia data here please?
The date that you have downloaded the data for your preparation.

Reply:
Yeah that's definitely required. Added!